FreeRTOS/FreeRTOS-Plus-TCP
# Fix udp ip failing unit test cases

Fixes UDP IP failing unit tests cases wrt IPv6 changes

Description
-----------
This PR fixes the failing test cases for the UDP IP unit tests.

Source file changes:

* source/FreeRTOS_UDP_IP.c:
Removed return value variable for function calls that doesn’t return anything 
Added return value for the functions that returns value.

Test Steps
-----------
```
cmake -S test/unit-test -B test/unit-test/build/ 
make -C test/unit-test/build/ all 
cd test/unit-test/build/
ctest -E system --output-on-failure --verbose

```

Related Issue
-----------
Failing test cases


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
